app/models/wpcc/your_details_form.rb

```diff
@@ -6,11 +6,16 @@ class YourDetailsForm
     attr_accessor :age, :gender, :salary
     attr_accessor :salary_frequency, :contribution_preference
 
+    AGE = { minimum: 16, maximum: 75 }.freeze
```

Comment:
Here it says max is 75, later it says max is 74. Please pick one number and stick to it. :)